### DISCUSSION TITLE
Unable to Push my update

### DISCUSSION THREAD

**Discussion by baynaj** [0x7c13/Notepads#1032] [General]
Hi @JasonStein, I have a PR for a feature (#1030 ), I am getting an error 403 when attempting to push the update.

**Comment by 0x7c13** [comment]
You cannot directly push any PR into the master branch of this main Repo. You can only push it to your branch and then raise a PR from your branch under your forked repo into mine.

**Reply by baynaj** [reply]
Thank you for your help. I’m working on a project for school and thought this would be a good addition to your application. Notepads is awesome btw! Sending a PR soon
